src/main/java/com/example/book_your_seat/common/entity/ErrorResult.java

```diff
@@ -0,0 +1,20 @@
+package com.example.book_your_seat.common.entity;
+
+import lombok.Data;
+
+@Data
+public class ErrorResult {
+
+    private String code;
+
+    private String message;
+
+    public ErrorResult(String code, String message) {
+        this.code = code;
+        this.message = message;
+    }
+
+    public ErrorResult(String code) {
+        this.code = code;
+    }
```

Comment:
```suggestion
    public ErrorResult(String code) {
        this.code = code;
        this.message = "bad request";
    }
```

이렇게 수정하시면 record 사용가능 할 듯 합니다!